### DISCUSSION TITLE
how to view all 'hidden items' on github issue page?

### DISCUSSION THREAD

**Discussion by comfortablemilk4454** [community/community#128155] [New to GitHub]
does anyone know how to expand all of the 1,317 hidden items on a github issue page? if i left click it it only expands a few more and id have to click like 100 times to see the whole conversation. trying to read through a recent message but it's one of the hidden ones. please refer to the screenshot.
![image](https://github.com/community/community/assets/43555540/f844ec9c-ede7-4dbe-b85a-6391ee225019)

the link to the issue page i experience this on is https://github.com/uBlockOrigin/uAssets/issues/20586.

my system information:
MacBook Pro 14" (early 2023, M2 pro apple silicon chip, 16g ram, 1tb storage)
macOS 15.0 Beta (24A5264n)
Brave Browser Version 1.66.118 Chromium: 125.0.6422.147 (Official Build) (arm64)

thanks so much in advance!

**Comment by buu-huu** [comment]
Hm, I don't know, if there is a dedicated feature from GitHub to do this, but the following workaround worked for me. Open the developer tools and run the following function in the console:

```javascript
(function clickLoadMoreButtons() {
    let loadMoreButton = document.querySelector('.ajax-pagination-btn');
    if (loadMoreButton) {
        loadMoreButton.click();
        setTimeout(clickLoadMoreButtons, 1000);  // Adjust timeout as needed
    }
})();
```

**Reply by thomas-j-wtfoxtrot** [reply]
In my case, I have added a
```
else alert('All PR threads loaded')
```

to get some feedback.

**Reply by Hi-Angel** [reply]
Doesn't work ☹ The class is `.prc-Button-ButtonBase-c50BI` now, however there're 64 elements with this class.

That said, if you're okay with pressing them all, here's a GreaseMonkey script that would automate this on visiting github:

```javascript
// ==UserScript==
// @name     Github: load all "hidden items"
// @version  1
// @require      https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js
// @require https://gist.githubusercontent.com/BrockA/2625891/raw/9c97aa67ff9c5d56be34a55ad6c18a314e5eb548/waitForKeyElements.js
// @match    https://github.com/*
// ==/UserScript==
const load_hidden_items = (node) => node.click();
waitForKeyElements('.prc-Button-ButtonBase-c50BI', load_hidden_items);
```

**Comment by comfortablemilk4454** [comment]
thanks for the code :)

**Reply by buu-huu** [reply]
No problem man :) If the code works for you, I would appreciate it, if you would mark my post as answer :)

**Reply by comfortablemilk4454** [reply]
its not the answer to the question in the post tho lol

**Reply by buu-huu** [reply]
Im pretty sure that there’s no feature to do this on GitHub, but let’s see if someone else has another solution ;)

**Comment by andry81** [comment]
You can a code lookup here: https://github.com/ludios/expand-everything

**Comment by marcindulak** [comment]
This problem is also present for discussions. I wonder whether anybody is able to follow what is happening on https://github.com/orgs/community/discussions/163932, there are 2.5k comments to expand there.

<img width="1308" height="318" alt="Screenshot from 2026-01-23 20-25-23" src="https://github.com/user-attachments/assets/c2e58931-0a47-40ca-8eef-f665f96919ee" />
